Wikipedia/UI-V5/WMFTableOfContentsCell.swift

```diff
@@ -0,0 +1,167 @@
+
+import UIKit
+
+// MARK: - Cell
+public class WMFTableOfContentsCell: UITableViewCell {
+
+    @IBOutlet var sectionSelectionBackground: UIView!
+    @IBOutlet var sectionTitle: UILabel!
+    @IBOutlet var selectedSectionIndicator: UIView!
+    @IBOutlet var indentationConstraint: NSLayoutConstraint!
+    @IBOutlet var leadSectionBorder: UIView!
+    @IBOutlet var sectionBorder: UIView!
+    
+    // MARK: - Init
+    public required init?(coder aDecoder: NSCoder) {
+        section = nil
+        super.init(coder: aDecoder);
+        self.indentationWidth = 10
+    }
+    
+    // MARK: - UIView
+    public override func awakeFromNib() {
+        super.awakeFromNib()
+        self.selectedSectionIndicator.alpha = 0.0
+        self.sectionSelectionBackground.backgroundColor = UIColor.wmf_tableOfContentsSelectionBackgroundColor()
+        self.sectionSelectionBackground.alpha = 0.0
+        self.selectionStyle = .None
+    }
+    
+    // MARK: - Accessors
+    public var section: MWKSection?{
+        didSet(section) {
+            self.sectionTitle.text = WMFTableOfContentsCell.titleText(self.section)
```

Comment:
We've been doing these kinds of transformations in the cell and sharing functionality via inheritance (see `setTitle(MWKTitle)` and `setImage(MWKImage)`. These kinds of transformations/logic are usually view-specific or inheritable, so unless it's going to be reused, abstraction isn't as much of a benefit.  I think it's fine as-is, but I think it would've been acceptably testable if the transformations were tested by setting a `MWKSection` and querying for expected values.